Resources/Prototypes/Body/Organs/slime.yml

```
@@ -49,19 +51,25 @@
           rateModifier: 0.25
+    - type: Extractable # goob edit
+      grindableSolutionName: organ # goob edit
     - type: SolutionContainerManager
       solutions:
         stomach:
           maxVol: 50.0
         food:
-          maxVol: 5
+          maxVol: 10 # goob edit
           reagents:
           - ReagentId: GreyMatter
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 12 but found 10

(indentation)